test/e2e_node/runner/remote/run_remote.go

```diff
@@ -907,6 +910,35 @@ func parseInstanceMetadata(str string) map[string]string {
 	return metadata
 }
 
+func crioSerialInjectSSHPublicKey(path string, content []byte) []byte {
+	// Exclude non CRI-O serial tests
+	if !strings.HasSuffix(path, "/test-infra/jobs/e2e_node/crio/crio_serial.ign") &&
+		!strings.HasSuffix(path, "/test-infra/jobs/e2e_node/crio/crio_cgrpv2_serial.ign") {
```

Comment:
This will only work on the serial jobs, but we have a number of other CRI-O presubmits we want to get working, and hardcoding one config path here is already kind of hacky, let alone 2. Can we factor out the constant and update this to

```suggestion
	if !strings.Contains(path, "/test-infra/jobs/e2e_node/crio/") {
```

Reply:
Looking this over, we might want to consider not restricting this to CRI-O, and instead checking an environment variable that we can configure in test-infra.

Reply:
Good, we can do that by using an env variable in https://github.com/kubernetes/test-infra/pull/25764